src/game/scenes/vs.c

```diff
@@ -232,24 +235,52 @@ void vs_render(scene *scene) {
     game_player *player1 = game_state_get_player(scene->gs, 0);
     game_player *player2 = game_state_get_player(scene->gs, 1);
 
-    font_render_shadowed(&font_small, local->vs_str, 160 - ((strlen(local->vs_str) * font_small.w) / 2), 3,
-                         COLOR_YELLOW, TEXT_SHADOW_RIGHT | TEXT_SHADOW_BOTTOM);
+    // X vs Y
+    text_settings tconf_yellow;
+    text_defaults(&tconf_yellow);
+    tconf_yellow.font = FONT_SMALL;
+    tconf_yellow.cforeground = COLOR_YELLOW;
+    tconf_yellow.shadow = TEXT_SHADOW_RIGHT | TEXT_SHADOW_BOTTOM;
+    tconf_yellow.cshadow = 202;
+    tconf_yellow.halign = TEXT_CENTER;
+
+    text_render(&tconf_yellow, TEXT_DEFAULT, 0, 3, 320, 8, local->vs_str);
+
+    // no other text is shadowed
+    tconf_yellow.shadow = 0;
 
     if(player2->selectable) {
+        text_settings tconf_green;
+        text_defaults(&tconf_green);
+        tconf_green.font = FONT_SMALL;
+        tconf_green.cforeground = TEXT_GREEN;
+        tconf_green.shadow = TEXT_SHADOW_RIGHT | TEXT_SHADOW_BOTTOM;
+        tconf_green.halign = TEXT_CENTER;
+        tconf_green.cshadow = TEXT_SHADOW_GREEN;
+
         // arena selection
         video_draw(&local->arena_select_bg, 55, 150);
 
         // arena name
-        font_render_wrapped(&font_small, lang_get(56 + local->arena), 56 + 72, 153, (211 - 72) - 4, TEXT_BRIGHT_GREEN);
+        text_render(&tconf_green, TEXT_DEFAULT, 56 + 72, 153, (211 - 72) - 4, 8, lang_get(56 + local->arena));
+        // font_render_wrapped(&font_small, lang_get(56 + local->arena), 56 + 72, 153, (211 - 72) - 4,
+        // TEXT_BRIGHT_GREEN);
 
+        tconf_green.valign = TEXT_MIDDLE;
         // arena description
-        font_render_wrapped(&font_small, lang_get(66 + local->arena), 56 + 72, 160, (211 - 72) - 4, TEXT_BRIGHT_GREEN);
+        text_render(&tconf_green, TEXT_DEFAULT, 56 + 72, 153, (211 - 72) - 4, 50, lang_get(66 + local->arena));
+        // font_render_wrapped(&font_small, lang_get(66 + local->arena), 56 + 72, 160, (211 - 72) - 4,
+        // TEXT_BRIGHT_GREEN);
```

Comment:
Could these old font_render stuffs be removed from the comments?

Reply:
I thought I got rid of all of them, sorry